### PR TITLE
Split stretchtests in stretchigt and stretchv4l2

### PR DESCRIPTION
**Pull request by ana**
The current stretchtests rootfs include two test suites:  v4l2 and IGT. This has lead to several problems:
    * the resulting rootfs are too big
    * sometimes we would like to build v4l2 more often than IGT
    * we don't need to build every rootfs for all the architectures
    
This PR rename stretchtests as stretchigt removing the v4l2 test suite, add a new job for v4l2 and make the corresponding changes in `test-configs.yaml`

### CODE REVIEW

**Comment by ana** [comment]
I tested this in staging-ci, please see:
https://staging-ci.kernelci.org/job/Debian%20stretch%20v4l2%20rootfs%20builder/
and
https://staging-ci.kernelci.org/job/Debian%20stretch%20igt%20rootfs%20builder/

**Comment by gctucker** [comment]
While looking at the other PR to add the "S" suite, and based on what we've done with the media subsystem, I started thinking that there should probably a clear link between the test plan name and the areas of the kernel being tested.  The name of the actual tests being used may be less useful, and we may want to run a mix of tests to cover a subsystem or a core component of the kernel.

While v4l2 is arguably a kernel feature, IGT isn't.  So, maybe we could call it the "drm-kms" test suite rather than igt?  We're really only testing this part of the kernel, not running the full IGT test suite (which I think would rather be part of an i915 test plan if we wanted to do that).  I'm not 100% sure which is best, but it seems like something worth considering at this point.

**Comment by ana** [comment]
> While looking at the other PR to add the "S" suite, and based on what we've done with the media subsystem, I started thinking that there should probably a clear link between the test plan name and the areas of the kernel being tested. The name of the actual tests being used may be less useful, and we may want to run a mix of tests to cover a subsystem or a core component of the kernel.
> 
> While v4l2 is arguably a kernel feature, IGT isn't. So, maybe we could call it the "drm-kms" test suite rather than igt? We're really only testing this part of the kernel, not running the full IGT test suite (which I think would rather be part of an i915 test plan if we wanted to do that). I'm not 100% sure which is best, but it seems like something worth considering at this point.

So while this PR is only about the rootfs used the test suites and it's not touching the test plan, I think this is a good point and we should discuss the naming of the test plans and the rootfs naming - since its name should point to the test plans using them.
Maybe we can add a point for the meeting on Monday?

**Comment by khilman** [comment]
The separating and renaming looks good to me, and I agree with @gctucker that adding a hyphen is easier on the eyes.

File: test-configs.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@ file_system_types:
     arch_map:
       armhf: {arch: arm}
 
+  debian_stretchv4l2:
+    url: 'http://staging-storage.kernelci.org/images/rootfs/debian/stretchv4l2/20181109.0'
```
Comment (gctucker) [inline test-configs.yaml]: Please update the URL to use `20181116.0` as it has the latest `v4l2-parser.sh` script.

PS: I know this is annoying, see discussion on IRC about having a "latest rootfs version" symlink on the storage server...

**Comment by ana** [comment]
I've updated the commits to use stretch-igt and stretch-v4l2. I've also rebuilt the images and updae `test-configs.yaml`  to use these images from today.

File: test-configs.yaml (hunk 1 above)
Comment (ana) [reply test-configs.yaml]: I have updated the link to use the images built today.

**Pull request merged**